cmd/antrea-agent/options.go

```diff
@@ -484,13 +480,6 @@ func (o *Options) setK8sNodeDefaultOptions() {
 	}
 
 	if features.DefaultFeatureGate.Enabled(features.Multicluster) {
-		if o.config.Multicluster.Enable {
```

Comment:
we should have had a log message about this being deprecated, but as you pointed out Multicluster is still Alpha so I am fine with this change.